packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -675,6 +675,14 @@ class _FocusState extends State<Focus> {
     Widget child = widget.child;
     if (widget.includeSemantics) {
       child = Semantics(
+        // Automatically request the focus for a focusable widget when it
```

Comment:
Shouldn't this be listening to onFocus callback instead?

Reply:
Yes, I hadn't updated it yet to match Yegor's change.